include/internal/punycode.h

```diff
@@ -0,0 +1,15 @@
+#ifndef HEADER_INTERNAL_PUNYCODE_H
```

Comment:
Copyright boilerplate missing.

Also, unless you think this is needed outside of libcrypto, please move this to `include/crypto/` and rename the `HEADER_INTERNAL_PUNYCODE_H` to something that's more like in the other files there.

Reply:
Done.